docs/source/tutorial/documentation.rst

```diff
@@ -379,8 +379,8 @@ We can add annotations to our code as follows.
 
     From `PEP 484 <https://www.python.org/dev/peps/pep-0484/#id27>`_:
 
-    *Rather than requiring that users write import numbers and then use numbers.Float etc.,
+    *Rather than requiring that users write ``import numbers`` and then use ``numbers.Float`` etc.,
```

Comment:
One of the annoying drawbacks of restructured text is that you can't combine styling. So the fact that this is emphasized precludes it from containing verbatim text (like teletype). 😱

Reply:
Uggggh, how silly ☹️